instauto/helpers/friendships.py

```diff
@@ -10,20 +10,30 @@
 logger = logging.getLogger(__name__)
 
 
-def get_followers(client: ApiClient, user_id: str, limit: int) -> typing.List[models.User]:
+def get_followers(client: ApiClient, user_id: str, username: str, limit: int) -> typing.List[models.User]:
     """Retrieve the first x amount of followers from an account.
 
-    If the `user_id` is not known, use `instauto.helpers.search.get_user_id_from_username`
-    to convert a username to user_id.
+    Either `user_id` or `username` need to be provided. If both are provided,
+    the user_id takes precedence.
 
     Args:
         client: your ApiClient
         user_id: the user_id of the account to retrieve followers from
+	    username: the username of the account to retrieve followers from
         limit: the maximum amount of followers to retrieve
 
     Returns:
         A list containing Instagram user objects (examples/objects/user.json).
     """
+    if user_id is not None and username is not None:
```

Comment:
If both are provided, we just use the `user_id`. It's like that everywhere, so let's keep that standard.

Reply:
Sure thing! I copied the argument checking logic from the functions further down in that file. Should I change it there too?

Reply:
Aha, if you could :) Good catch